tests/util/base.py

```diff
@@ -135,32 +135,52 @@ def make_default_account(db, config):
     return account
 
 
-@fixture(scope='function')
+def delete_all_namespaces_and_authcreds(db):
+    from inbox.models.backends.gmail import GmailAccount
+    from inbox.models.backends.gmail import GmailAuthCredentials
+    from inbox.models import Namespace
+    delete_messages(db.session)
+    db.session.rollback()
+    db.session.query(GmailAccount).delete()
+    db.session.query(GmailAuthCredentials).delete()
+    db.session.query(Namespace).delete()
+    db.session.commit()
+
+
+@yield_fixture(scope='function')
 def default_account(db, config):
-    return make_default_account(db, config)
+    yield make_default_account(db, config)
```

Comment:
Would be nice if `make_default_account` and the corresponding `delete_**` had matching names, so you knew one undid the other. This looks 👍  though.